pkg/userinterface/userinterface.py

```diff
@@ -1,44 +1,92 @@
 import PySimpleGUI as sg
+
```

Comment:
Looks like my formatter touched this up. Are you ok if I just leave these changes in? They're purely style changes.

FWIW I'm using `black` (`pip install black`) if you want to use the same one

Reply:
Definitely cool with that; I followed the formatting from PySimpleGUI's own demo code, but I felt like it looked rough. I'll incorporate that formatter into my code too.

Reply:
It wasn't clear to me at first but running `black pkg` will format only our package code (instead of all the included libraries which takes a while and is unwanted)